routes/movies.routes.js

```diff
@@ -0,0 +1,95 @@
+// starter code in both routes/celebrities.routes.js and routes/movies.routes.js
+const router = require("express").Router();
+const Movie = require("../models/Movie.model.js");
+const Celebrity = require("../models/Celebrity.model");
+// all your routes here
+
+router.get("/movies/create",async (req,res) => {
+    let allCelebritiesFromDb = await Celebrity.find();
+
+    res.render("movies/new-movie.hbs", {celebrities: allCelebritiesFromDb});
+});
+
+
+router.post("/movies/create", async (req,res) => {
+    try {
+        const {title, genre, plot, cast} = req.body;
+        
+        await Movie.create({title, genre, plot, cast});
+        res.redirect("/movies");
+    }
+    catch (error){
+        console.log(error)
+        res.render("movies/new-movie.hbs");
+    }
+});
+
+
+router.get("/movies", async (req,res) => {
+    try{
+        let allMoviesFromDb = await Movie.find();
+
+        res.render("movies/movies.hbs", {movies: allMoviesFromDb});
+
+    }
+    catch(error){
+        console.log("Error while getting movies", error);
+    }
+});
+
+
+router.get("/movies/:movieId", async (req,res) => {        
+    try{
+        const {movieId} = req.params;
+
+        let chosenMovie = await Movie.findById(movieId).populate("cast");
+        
+        res.render("movies/movie-details.hbs", {chosenMovie});
```

Comment:
Since you are doing a `findById()` you receive just one object property, so, in this case, you don't need to send `chosenMovie` inside "{ }".
`res.render("movies/movie-details.hbs", chosenMovie);`